lib/Common/CommonPal.h

```diff
@@ -274,6 +274,10 @@ inline T InterlockedDecrement(
     return __sync_sub_and_fetch(Addend, T(1));
 }
 
+inline __int64 _abs64(__int64 n)
```

Comment:
Isn't that (_abs64) already part of standard library? (MS flavored)
BTW; If I'm not mistaking, llabs, abs are already used in the project.